src/NzbDrone.Core/Download/TrackedDownloads/DownloadMonitoringService.cs

```diff
@@ -128,12 +144,12 @@ public void Execute(CheckForFinishedDownloadCommand message)
             Refresh();
         }
 
-        public void HandleAsync(EpisodeGrabbedEvent message)
+        public void Handle(EpisodeGrabbedEvent message)
```

Comment:
Won't this cause any callers to pause and wait since Handle executes in series, not parallel?